client/cypress/e2e/grid-component.cy.ts

```diff
@@ -8,12 +8,12 @@ describe('Grid Component', () => {
     cy.task('seed:database');
   });
 
-  it('should open saved grid to server', () => {
-    cy.get(':nth-child(2) > .ng-star-inserted > button').click();
-    cy.get(':nth-child(2) > .ng-star-inserted > button').click();
```

Comment:
Was there a purpose to the second `cy.get` line (line 13)? That line causes e2e to fail on my computer, but if commented out, it passes. Have you looked into this second line being the reason GitHub might be failing?